### DISCUSSION TITLE
Mod tier functionality

### DISCUSSION THREAD

**Discussion by Leroy8888** [HelixNebulaStudio/RiseOfTheDead#38] [General]
I am aware that this function is not yet final, but I was curious to what the reason for its implementation was. It seems rather odd to reverse weapon mod tier compatibility while the system seems to work fine.

Not looking at the balance of weapons or mods as that will likely be adjusted. But seeing that tempering with mechanics like these might upset a large portion of the community as existing builds become weaker or irrelevant, and thus have to spend more perks to adjust it. 

It might be important to discuss about the inclusion of the mechanic before too much effort and time is spend into it, as it could end up not being desired by the community and cause more harm then good.

**Comment by MXKhronos** [comment]
The reverse for the tier compatibility was introduced because I was thinking to add a way for mods to be compatible with the weapon tiers and in order to do that, the compatibility had to be reversed. This will also result in all tier 1 damage mods inferior on higher tier weapons which essentially nerfs them too. The whole idea was to introduce a additional cost for e.g. going from a tier 1 pistol to tier 2 pistol, you will need to upgrade the mods too. I was also considering whether tier compatibility should be exact match instead for full compatibility.

But based on your points and after a while of reconsiderations, I do realize my initial point for tier mods was limit the strength of a mod to a range of weapon tiers so a low tier mod wouldn't be as dominating in the meta and limit the feasible build variations.

I may either find a compromise or undo the changes.

**Reply by MXKhronos** [reply]
Testing mark answered
